src/components/Table/GridTable.test.tsx

```diff
@@ -2110,6 +2111,19 @@ describe("GridTable", () => {
     expect(cell(r, 1, 1)).toHaveStyleRule("background-color", Palette.LightBlue50);
   });
 
+  it("reacts to setting activeCellId", async () => {
+    // Given a table initially rendered with an active cell id
+    const r = await render(
+      <GridTable columns={columns} rows={rows} style={{ cellHighlight: true }} activeCellId={"data_1_name"} />,
+    );
+    // Then expect the cell to have highlight style
+    expect(cell(r, 1, 0)).toHaveStyleRule("border-width", "1px");
+    expect(cell(r, 1, 0)).toHaveStyleRule("border-color", Palette.LightBlue900);
+    // Then expect other cells not to have highlight style
+    expect(cell(r, 1, 1)).not.toHaveStyleRule("border-width", "1px");
+    expect(cell(r, 1, 1)).not.toHaveStyleRule("border-color", Palette.LightBlue900);
```

Comment:
**question:** Shouldn't the other cells have the border style, but with a transparent color?